galeracluster/source/installmysqlsrc.rst

```diff
@@ -6,65 +6,166 @@ Source Installation
+
+   .. code-block:: console
 
-The source code for Galera Cluster for MySQL is available through `Github <https://github.com/codership/mysql-wsrep>`_.  You can download the source code using ``git``.
+      # git clone https://github.com/codership/mysql-wsrep
+
+#. Checkout the branch for the version that you want to use.
+
+   .. code-block:: console
+
+      # git checkout 5.6
+
+   The main branches available for Galera Cluster for MySQL are:
+
+   - ``5.6``
+   - ``5.5``
+
+  
+You now have the source files for the MySQL database server, including the wsrep API patch needed for it to function as a Galera Cluster node.
+
+In addition to the database server, you need the wsrep Provider, also known as the Galera Replication Plugin.  In a separator directory, run the following command:
 
 .. code-block:: console
 
-   $ git clone https://github.com/codership/mysql-wsrep.git
+   # cd ..
+   # git clone https://github.com/codership/galera.git
 
-Once ``git`` finishes running, you can start building the database server.  You have two options for how to build Galera Cluster for MySQL.  You can use the build script, or you can build it using ``cmake``.  If you choose the build script, the command will vary depending upon your system architecture.  From the ``mysql-wsrep/`` directory, use one of the following commands:
+Once Git finishes downloading the source files, you can start building the database server and the Galera Replication Plugin.  The above procedures created two directories: ``mysql-wsrep/`` for the database server source and for the Galera source ``galera/``
 
-- To run the build script on a 64-bit system, use this command:
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Building the Database Server
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. _`build-mysql`:
 
-  .. code-block:: console
+The database server for Galera Cluster is the same as that of the standard database servers for  standalone instances of MySQL, with the addition of a patch for the wsrep API, which is packaged in the version downloaded from GitHub_.  You can enable the patch through  the wsrep API, requires that you enable it through the ``WITH_WSREP`` and ``WITH_INNODB_DISALLOW_WRITES`` CMake configuration options.
 
-     # BUILD/compile-pentium64
+To build the database server, ``cd`` into the ``mysql-wsrep/`` directory and run the following commands:
 
-- To run the build script on a 32-bit system, instead use this command:
+.. code-block:: console
 
-  .. code-block:: console
+   # cmake -DWITH_WSREP=ON -DWITH_INNODB_DISALLOW_WRITES=ON ./
+   # make
+   # make install
 
-     # BUILD/compile-penitum
+.. note:: In addition to compiling through ``cmake`` and ``make``, there are also a number of build scripts in the ``BUILD/`` directory, which you may find more convenient to use.  For example, 
 
-- To build Galera Cluster for MySQL using ``cmake``, run the following commands:
+	  .. code-block:: console
 
-  .. code-block:: console
+	     # ./BUILD/compile-pentium64
```

Comment:
I think they used to, at least on 5.5. But I don't those are relevant any more, you are right.